media/css/custom.scss

```diff
@@ -0,0 +1,468 @@
+// Core variables and mixins
+@import "variables";
+@import "mixins";
+
+html,
+body {
+  height: 100%;
```

Comment:
Properties should be ordered background, color, font-size, height, text-align, text-shadow